cpp/src/parquet/encoding.h

```diff
@@ -296,13 +296,16 @@ class TypedDecoder : virtual public Decoder {
 
     // Depending on the number of nulls, some of the value slots in buffer may
     // be uninitialized, and this will cause valgrind warnings / potentially UB
+    // TODO: With SIMD shuffle in the loop below we might be able to get better
```

Comment:
Instead of an in-code TODO, how about a JIRA?

Reply:
Good idea.  https://issues.apache.org/jira/browse/PARQUET-1841

@jianxind I added comments below and this JIRA if you still want to investigate this and need more context I'm happy to discuss on the PARQUET-1841 JIRA.